package.json

```diff
@@ -50,6 +53,7 @@
     "mongodb": "^3.6.6",
     "morgan": "^1.9.1",
     "nocache": "^2.1.0",
+    "nodemon": "^2.0.16",
```

Comment:
should this be a dev dependency?

Reply:
Good catch. 

[89cb808](https://github.com/AMP-SCZ/dpdash/pull/71/commits/89cb8080debb06653b9dca6dce4f49a2bb6260a8)